Containers/mastercontainer/supervisord.conf

```
@@ -8,14 +8,22 @@ logfile_backups=10
 loglevel=error
 user=root
 
-[program:apache]
+[program:php-fpm]
 # stdout_logfile=/dev/stdout
 # stdout_logfile_maxbytes=0
 stderr_logfile=/dev/stderr
 stderr_logfile_maxbytes=0
-command=apache2-foreground
+command=php-fpm
 user=root
```

maybe also use www-data user?
```suggestion
user=www-data
```

---

yeah, we could try that 👍